src/mammos_entity/operations.py

```diff
@@ -21,6 +21,7 @@
 def concat_flat(
     *elements: mammos_entity.typing.EntityLike | list[typing.Any] | tuple[typing.Any],
     unit: astropy.units.Unit | str | None = None,
+    description: str | None = None,
 ) -> mammos_entity.Entity:
     """Concatenate objects into a unique flat Entity.
 
```

Comment:
- We need to document how we deal with `description` of the individual elements.
- I am not sure if the current implementation is our preference. Maybe we should retain a description from an entity if `description=None`?